src/systems/lift_drag/LiftDrag.hh

```diff
@@ -38,7 +38,8 @@ namespace systems
   /// The following parameters are used by the system:
   ///
   /// link_name   : Name of the link affected by the group of lift/drag
-  ///               properties.
+  ///               properties. This can be a scoped name to reference links in
```

Comment:
What do you think about adding `\sa entitiesFromScopedName` for users who do not know what scoped names are?

Reply:
Good idea. https://github.com/ignitionrobotics/ign-gazebo/pull/955/commits/aa26825053a65c20700291b9b988ff12de37704e